src/pp/ppcore.c

```diff
@@ -5747,6 +5759,10 @@ void hb_pp_initDynDefines( PHB_PP_STATE pState, HB_BOOL fArchDefs )
 #ifdef HB_START_PROCEDURE
    hb_pp_addDefine( pState, "__HB_MAIN__", HB_START_PROCEDURE );
 #endif
+
+   if(fDebug)
```

Comment:
Please note, that a valid pull request should also have usual code style/formatting.
```c
if( fDebug )
```
Here it should be with spaces. Another example few lines back.